frontend/pages/reports.js

```diff
@@ -28,6 +28,29 @@ const serializeParams = params => {
     .join('&')
 }
 
+const HoverListElement = s.li`
+  span {
+    display: none;
+  }
+  &:hover span {
+    display: inline;
+    cursor: pointer;
+  }
```

Comment:
This doesn't really work with mobile—do we care about that use case?

Also could we maybe try like `margin-left: auto` so all the icons will be aligned on the right side of the sidebar? Might be better UI